spark3-extensions/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/ReplaceData.scala

```diff
@@ -25,4 +25,12 @@ import org.apache.spark.sql.connector.write.BatchWrite
 case class ReplaceData(
     table: NamedRelation,
     write: BatchWrite,
-    query: LogicalPlan) extends V2WriteCommand
+    query: LogicalPlan) extends V2WriteCommand {
+
+  def isByName: Boolean = false
```

Comment:
I am not sure if we want this to be true or false. I suspect it doesn't really matter.

Reply:
It shouldn't matter.

Reply:
Ok, thanks.